vrouter.go

```diff
@@ -300,11 +307,87 @@ func (self *Vrouter) AddLocalEndpoint(endpoint OfnetEndpoint) error {
 		}
 	}
 
+	if endpoint.HostPvtIP.String() != "<nil>" && self.hostNATInfo.PortNo != 0 {
```

Comment:
Based on the documentation, it does not seem to be the same thing. It it's uninitialized, String will return <nil>.